packages/shared/src/util.ts

```diff
@@ -21,6 +21,20 @@ export function titleizeImageryName(name: string): string {
   );
 }
 
+/**
+ * Make a tile imagery title as imagery name
+ * @example
+ *  'Tasman rural 2018-19 0.3m' => 'tasman_rural_2018-19_0-3m'
+ */
+export function nameImageryTitle(title: string): string {
+  return title
+    .replace(/ /g, '_')
+    .replace('.', '-')
+    .replace(/[^a-zA-Z0-9-_]/g, '')
```

Comment:
I wonder if there is a way to preserve the utf8 macrons here, eg `Manawatū`

Reply:
Not sure yet. I will make a test for this and see how to keep it. Might just working.

Reply:
Add `À-ȕ` in the filter to keep these macrons.

Reply:
How far does that reach? will this will cover all macrons used in NZ?

Reply:
![image](https://user-images.githubusercontent.com/12163920/206571137-388d4d08-854f-40eb-92c0-43fde71d4127.png)
I think it covers all the `a e i o u` I think.